Tests/Xcodeproj/FunctionalTests.swift

```diff
@@ -21,13 +21,55 @@ class FunctionalTests: XCTestCase {
             XCTAssertXcodeprojGen(prefix)
             let pbx = Path.join(prefix, "Library.xcodeproj")
             XCTAssertDirectoryExists(pbx)
-            // FIXME: Move this to a method.
-            try popen(["env", "-u", "TOOLCHAINS", "xcodebuild", "-project", pbx])
+            XCTAssertXcodeBuild(project: pbx)
```

Comment:
I forget the details of how these fixtures are written, where is this actually building?